event_sale_session/reports/sale_report.py

```diff
@@ -27,13 +27,12 @@ class SaleReport(models.Model):
         readonly=True,
     )
 
-    def _select_sale(self):
-        select = super(SaleReport, self)._select_sale()
-        select += (
-            ", l.event_id as event_id, l.event_ticket_id as event_ticket_id, "
-            "l.event_session_id as event_session_id"
-        )
-        return select
+    def _select_additional_fields(self):
+        res = super()._select_additional_fields()
+        res['event_id'] = 'l.event_id'
+        res['event_ticket_id'] = 'l.event_ticket_id'
+        res['event_session_id'] = 'l.event_session_id'
+        return res
```

Comment:
Technically it looks good, but we do need to get pre-commit to pass.

Could you apply the changes suggested by it, in the ci:

```diff
diff --git a/event_sale_session/reports/sale_report.py b/event_sale_session/reports/sale_report.py
index e7868e4..6a6f937 100644
--- a/event_sale_session/reports/sale_report.py
+++ b/event_sale_session/reports/sale_report.py
@@ -29,9 +29,9 @@ class SaleReport(models.Model):
 
     def _select_additional_fields(self):
         res = super()._select_additional_fields()
-        res['event_id'] = 'l.event_id'
-        res['event_ticket_id'] = 'l.event_ticket_id'
-        res['event_session_id'] = 'l.event_session_id'
+        res["event_id"] = "l.event_id"
+        res["event_ticket_id"] = "l.event_ticket_id"
+        res["event_session_id"] = "l.event_session_id"
         return res
 
     def _group_by_sale(self):
```

Reply:
Let me check if the template is not the latest one in this repository.